Doc/library/datetime.rst

```diff
@@ -589,8 +589,8 @@ Supported operations:
 +-------------------------------+----------------------------------------------+
 | Operation                     | Result                                       |
 +===============================+==============================================+
-| ``date2 = date1 + timedelta`` | *date2* is ``timedelta.days`` days removed   |
-|                               | from *date1*. (1)                            |
+| ``date2 = date1 + timedelta`` | *date2* will be ``timedelta.days`` days      |
```

Comment:
Why change `is` to `will be`?

Reply:
That was the documentation that @ericvsmith and @akulakov agreed upon in the original PR (which for some reason was never pulled). It does seem more consistent to use "is," should I change it to that?

Reply:
In my mind I read it as "After executing this statement, date2 will be ...".

But I don't have a strong preference one way or the other. If you think "is" is better, then change it.

Reply:
I do think "will be" fits better in this case since this is an assignment operation, where date2 *will* have a new value after the operation. The other case where "is" is used in the surrounding documentation is a comparison operation, where date2 and date1 *are* values. The usage fits with ordinary language, and should therefore have less of a chance of causing confusion.

If @merwok doesn't have strong objections, my vote is also to go with "will be" for this case.

Reply:
Let's leave it as "will be".